scripts/release.sh

```diff
@@ -342,13 +342,15 @@ main() {
     local release_url
     local gh_release_args=()
 
-    # For the main branch (v3.6), we should mark the release as a prerelease.
-    # The release-3.5 (v3.5) branch, should be marked as latest. And release-3.4 (v3.4)
-    # should be left without any additional mark (therefore, it doesn't need a special argument).
-    if [ "${BRANCH}" = "main" ]; then
-      gh_release_args=(--prerelease)
-    elif [ "${BRANCH}" = "release-3.5" ]; then
-      gh_release_args=(--latest)
+    local prompt
+    read -p "Mark this as a the latest release [y/N]? " -r prompt
```

Comment:
Can you leave a note that we expect that latest releases come from latest branch and not from patches to old releases?

Reply:
Ah, thanks for the review and comment, @serathius. You caught me before I undrafted this pull request :smile:. I'll be sure to apply the suggestion, I'm still working on this.

Reply:
Done